week1/index.js

```diff
@@ -1,10 +1,86 @@
-// To run this file: node index.js
-// Then in your browser: http://localhost:8080
-var http = require('http');
+// Declare global variables
+const http = require('http');
+//start server 
+const server = http.createServer();
+//This port was already chosen by the assignment maker.
+const port = 8080;
+//Object containing the respons text and the content type.
+//this response is the basic response.
+//it will be passed around by functions and modified untils it's in shape to be returned to the client
+let responsToSendBack = {
+    code:'<html></html>',
+    type:'text/html',
+    statusCode: 200,
+    state: Number
+};
 
-var port = 8080;
+//the original state which will always be 10. 
+//not allowed to change so it's a constant
+const originalState = 10;
```

Comment:
Great use of constant. Defaults are a perfect use-case for `const`.

A convention you'll sometimes see is the use of capital letters for constants, like `const ORIGINAL_STATE = 10`. This helps make a visual distinction between variable names and constant names.